### ISSUE TITLE
airframe-http requires airframe-di-macros from 20.12.2

### ISSUE THREAD

**Issue by yoyama** [wvlet/airframe#1417]
I updated airframe version from `20.12.1` to `20.12.2` and it caused build error as follows.

```
[error] /github/workspace/digdag-client/src/test/scala/io/github/yoyama/digdag/client/api/ApiDigdagServerFixture.scala:35:26: macro implementation not found: registerTraitFactory
[error] (the most common reason for that is that you cannot use macro implementations in the same compilation run that defines them)
[error]   def router = Router.add[DigdagApi]
```
As a workaround, I explicitly added 'airframe-di-macros`  in the dependency and fix it.
But I am not sure it is correct way or not.

https://github.com/yoyama/digdag-client-scala/runs/1636208339?check_suite_focus=true#step:6:186

**Comment by xerial** [comment]
Thanks for the report. This was an issue in build.sbt. I'll release 21.1.0 with a fix for it.

**Comment by xerial** [comment]
Released 21.1.0 with this fix
